src/services/github.ts

```diff
@@ -61,14 +61,12 @@ export default class GithubService {
     return searchResults.map((r) => _.get(r, 'full_name')).sort();
   }
 
-  private async findReposByCode(criteria: RestEndpointMethodTypes['search']['code']['parameters']): Promise<string[]> {
-    const searchResults: RestEndpointMethodTypes['search']['code']['response']['data']['items'] =
-    await this.paginateRest(this.octokit.search.code, criteria);
-
-    return searchResults.map((r) => _.get(r, 'repository.full_name')).sort();
+  private async findReposByCode(criteria: RestEndpointMethodTypes['search']['code']['parameters']):
+  Promise<RestEndpointMethodTypes['search']['code']['response']['data']['items']> {
+    return this.paginateRest(this.octokit.search.code, criteria);
   }
 
-  private getRepos(criteria: RestEndpointMethodTypes['repos']['get']['parameters']):
+  private getRepo(criteria: RestEndpointMethodTypes['repos']['get']['parameters']):
```

Comment:
AFAICT this only gets a single repo, so this name seemed more fitting.

Reply:
Checked the Octokit docs and it seems that's the case (see [1]).

References
1. https://octokit.github.io/rest.js/v18#repos-get